zerver/lib/markdown/__init__.py

```diff
@@ -2220,12 +2220,12 @@ def build_inlinepatterns(self) -> markdown.util.Registry:
         reg = markdown.util.Registry()
         reg.register(BacktickInlineProcessor(markdown.inlinepatterns.BACKTICK_RE), "backtick", 105)
         reg.register(
-            markdown.inlinepatterns.DoubleTagPattern(STRONG_EM_RE, "strong,em"), "strong_em", 100
+            Tex(r"\B(?<!\$)\$\$(?P<body>[^\n_$](\\\$|[^$\n])*)\$\$(?!\$)\B", self), "tex", 103
         )
-        reg.register(UserMentionPattern(mention.find_mentions, self), "usermention", 95)
```

Comment:
I don't think this change belongs in the Tex priority change commit.

Reply:
I'll do a clean push, resolving all the conflicts.